src/api/app/models/bs_request_action.rb

```diff
@@ -287,8 +287,14 @@ def notify_params(ret = {})
   end
 
   def contains_change?
-    return sourcediff.present?
-  rescue BsRequestAction::DiffError
+    source_info_target = Backend::Api::Sources::Package.source_info(target_project, target_package, nofilename: 1)
+    source_info_source = Backend::Api::Sources::Package.source_info(source_project, source_package, nofilename: 1)
+
+    hash_sum_target = Xmlhash.parse(source_info_target)['verifymd5']
+    hash_sum_source = Xmlhash.parse(source_info_source)['verifymd5']
+
+    hash_sum_target && hash_sum_target == hash_sum_source
```

Comment:
Wouldn't it be ``hash_sum_target != hash_sum_source`` ? Because if the hashes are the same, there is NO change and I would expect ``contains_change?`` to return false? (or maybe I got it wrong ...)

Reply:
:blush: ouch